torch/_inductor/scheduler.py

```diff
@@ -1828,9 +1899,11 @@ def can_eliminate_user(user: NodeUser) -> bool:
         for node in self.nodes:
             node.prune_weak_deps()
 
-    def topological_sort_schedule(self) -> None:
+    def topological_sort_schedule(
```

Comment:
By passing `nodes` into topological_sort_schedule / fuse_nodes / fuse_nodes_once / etc., we allow specifying which nodes we want to toposort or find fusion within.